src/main/resources/db/migration/V1_61__csip_plan.sql

```diff
@@ -0,0 +1,8 @@
+alter table csip_plan_mapping add column
+    dps_csip_report_id      varchar(64) not null;
+
+alter table csip_plan_mapping add constraint
+    dps_csip_report_id_fk1 FOREIGN KEY (dps_csip_report_id)
+    REFERENCES csip_mapping (dps_csip_id);
+
+create index csip_plan_mapping_dps_csip_report_id_index ON csip_plan_mapping (dps_csip_report_id);
```

Comment:
very minor, but I would have naturally put the nomis id on the table rather than DPS one but I'm not sure why :-)

Reply:
I thought that originally, but then because the delete pattern is via dpsId, I used that to make the calls simpler.